conftest.py

```diff
@@ -7,13 +8,16 @@
 
 import pytest
 import s3fs
+from dask.distributed import Client
 
 try:
     from coiled.v2 import Cluster
 except ImportError:
     from coiled._beta import ClusterBeta as Cluster
 
-from dask.distributed import Client
+
+# So coiled logs can be displayed on test failure
+logging.getLogger("coiled").setLevel(logging.INFO)
```

Comment:
This worked locally for me. @shughes-uk @ntabris suggestions are welcome if there's a more recommended way of doing this

Reply:
So we have a new failure but I can't seem to find the logs on CI. 
https://github.com/coiled/coiled-runtime/runs/6135051159?check_suite_focus=true

Reply:
Looks like it didn't even get as far as connecting to cloud.coiled.io so there wouldn't be logs about creating cluster. Did DNS break in your CI or something? You might just try running again and see if anything changes.

```
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host cloud.coiled.io:443 ssl:default [nodename nor servname provided, or not known]
```

Reply:
For pytest you'll ned to configure a logging handler that outputs to stdout 

https://realpython.com/python-logging/#basic-configurations this is a nice fast way to add a logger for that

Reply:
You can also run pytest with ```pytest --log-cli-level INFO` , but be warned this will set ALL loggers to `INFO` level